compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -259,7 +258,7 @@ pub enum GlobalAlloc<'tcx> {
     /// The alloc ID is used as a function pointer.
     Function { instance: Instance<'tcx> },
     /// This alloc ID points to a symbolic (not-reified) vtable.
```

Comment:
```suggestion
    /// This alloc ID points to a symbolic (not-reified) vtable.
    /// We remember the full dyn type, not just the principal trait, so that const-eval and
    /// Miri can detect UB due to invalid transmutes of `dyn Trait` types.
```